.gitignore

```diff
@@ -9,3 +9,6 @@
 coverage.txt
 profile.out
 /.glide
+
+# files generated during tests
+config/temp
```

Comment:
I don't mind removing but one thing is if viewing git diff during a test run, it shows many uncommitted files before this. I think that could be a good reason to leave it in. I'm using some Atom extensions that mean even just saving will run the tests.